src/System.Console/tests/ConsoleEncoding.cs

```diff
@@ -11,6 +11,8 @@
 
 public partial class ConsoleEncoding : RemoteExecutorTestBase
 {
+    public static bool ConsoleEncodingAvailable => !PlatformDetection.IsUap || PlatformDetection.IsWindows10RS3OrGreater;
```

Comment:
Does this actually work on RS3 for UAP (ie do you have a box to check this)?

I think I missed why RS3 is special here - I didn't know they did any work in RS3 for this scenario save expose the API in WACK which again isn't relevant to our tests. @JeremyKuhne ?